proxy-node-gateway/src/main/java/uk/gov/ida/notification/redis/storage/StorageType.java

```diff
@@ -0,0 +1,12 @@
+package uk.gov.ida.notification.redis.storage;
+
+import uk.gov.ida.notification.session.GatewaySessionData;
+
+public interface StorageType {
```

Comment:
It isn't very clear to me what a `StorageType` should be if I just saw that a class implements it. Maybe something like `SessionStore` would be a bit more reflective of what it is actually doing.